Formula/elixir-ls.rb

```diff
@@ -0,0 +1,34 @@
+class ElixirLs < Formula
+  desc "Language Server and Debugger for Elixir"
+  homepage "https://elixir-lsp.github.io/elixir-ls"
+  url "https://github.com/elixir-lsp/elixir-ls/archive/refs/tags/v0.9.0.tar.gz"
+  sha256 "eaa2f5fd4ab9e497889e25419e4409cfaf808daca06259718668a71de8616e3f"
+  license "Apache-2.0"
+
+  depends_on "elixir"
+
+  ENV["MIX_ENV"] = "prod"
+
+  def install
+    system "mix", "local.hex", "--force"
+    system "mix", "local.rebar", "--force"
+    system "mix", "deps.get"
+    system "mix", "compile"
+    system "mix", "elixir_ls.release", "-o", prefix.to_s
+  end
+
+  def caveats
+    <<~EOS
+      By default, elixir-ls will be placed into:
```

Comment:
Why not install it into `bin`?

Reply:
It's not a single binary - the artefact is a set of archives and shell script.

Reply:
Ok, but that doesn't really explain why we can't install executables (or symlinks/shim scripts to them) in the `bin` directory.

Reply:
Is there an example formula / variable that specifies the `bin` directory? I was looking in the handbook but never found it.

Reply:
We don't want to specify the bin directory to point to the install. We want to install binaries (or references to them) into the bin directory. See for example https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/alluxio.rb#L29-L31 installing everything to `libexec` and then setting up a wrapper in `bin`

Reply:
Hmm it's still not very clear to me what I'm supposed to do, in the [Formula Cookbook](https://docs.brew.sh/Formula-Cookbook) there is a brief explanation of the Formula variables, but not what the methods actually do or why, so I'm shooting a bit in the dark 🙂

Basically I produce a directory with some scripts that I want to be symlinked to `usr/local/bin`, that's all I need. In `    system "mix", "elixir_ls.release", "-o", prefix/"output"` I can specify where the output goes, but what should I exactly do after that? Sorry for the many questions, it's never really explained well in the cookbook.

Reply:
The cookbook doesn't reference this because it's more of a [general Unix software concept](https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard), it isn't specific at all to brew. My suggestion would be to do as the formula I linked above. Install everything into libexec and link use `bin.install` to move the binary files.

Reply:
Ok I figured it out, now it should be as per your suggestion, thanks.